src/batch-transaction.ts

```diff
@@ -20,12 +20,11 @@ import * as extend from 'extend';
 import * as is from 'is';
 import {Snapshot} from './transaction';
 import {google} from '../protos/protos';
-import {Session, Database} from '.';
+import {Session, Database, Spanner} from '.';
```

Comment:
nit: the Spanner import is no longer needed, now that we have moved the logic into a separate function (the `_getSpanner()` function)